examples/cognito/groupusermembership.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: cognitoidentityprovider.aws.crossplane.io/v1alpha1
+kind: GroupUserMembership
+metadata:
+  name: example-group-user-membership
+spec:
+  forProvider:
+    region: us-east-1
+    userPoolId: us-east-1_XXXXXXXX
```

Comment:
Can we use ref or selector in example ?

Reply:
I've updated the example to use `userPoolIdRef` and `groupnameRef`.